### ISSUE TITLE
GiveWP 2.9.0 should be compatible with WP 4.9.x

### ISSUE THREAD

**Issue by kevinwhoffman** [impress-org/givewp#5415]
## User Story

As an admin of a site on WP 4.9.15, I want GiveWP 2.9.0 to be compatible so that it does not crash my site due to a fatal error.

## Details

There are two known errors to look into:

### 1. Fatal error due to block registration

Fix this one first since it prevents any screen from loading.

![image](https://user-images.githubusercontent.com/1938671/97463118-cd0b2780-1915-11eb-9f12-4f7f1e6ae074.png)

### 2. Fatal error on onboarding wizard

Once you fix the error regarding block registration, make sure that the onboarding wizard does not throw an error.

![image](https://user-images.githubusercontent.com/1938671/97463218-e318e800-1915-11eb-8771-534b5b930ec6.png)

## Expected Behavior

The onboarding wizard and the plugin in general do not cause fatal errors on WP 4.9.x.

## Steps to Reproduce

1. Install WP 4.9.15 (there are [instructions for installing older WP versions in Local](https://localwp.com/community/t/install-old-wordpress-version/6982))
2. Install GiveWP 2.9.0
3. Observe the fatal error

## Acceptance Criteria

-   [x] GiveWP does not throw a fatal error due to block registration.
-   [x] With WP 4.9.x, a fresh install does not initiate the Onboarding Wizard so as to prevent a fatal error.
-   [x] With WP 4.9.x, the `Setup` menu item is not displayed in the admin menu.
-   [x] With WP 4.9.x, the `Setup Page` setting is not displayed in Donations > Settings > Advanced.
-   [x] The required version of WP is bumped from 4.8 to 4.9 in `readme.txt`.

**Comment by kevinwhoffman** [comment]
@henryholtgeerts Take note of the second fatal error that occurs on the Onboarding Wizard. I came across this error on a fresh install of WP 4.9.15 and GiveWP 2.8.1. Since the onboarding wizard gets automatically loaded for a fresh install, that error was the first thing I saw. The rest of the plugin still worked on 2.8.1.

When I upgraded GiveWP from 2.8.1 to 2.9.0, then I got the fatal error regarding block registration which prevented any screen from loading.
